gapic/templates/scripts/fixup_%service_keywords.py.j2

```diff
@@ -0,0 +1,125 @@
+{% extends '_base.py.j2' %}
+{% block content %}
+import argparse
+import os
+import libcst as cst
+from typing import (Any, Callable, Dict, List, Sequence, Tuple)
+
+
+def partition(
+    predicate: Callable[[Any], bool],
+    iterator: Sequence[Any]
+) -> Tuple[List[Any], List[Any]]:
+    """A stable, out-of-place partition."""
+    results = ([], [])
+
+    for i in iterator:
+        results[int(predicate(i))].append(i)
+
+    # Returns trueList, falseList
+    return results[1], results[0]
+
+
+class {{ service.client_name }}CallTransformer(cst.CSTTransformer):
+    CTRL_PARAMS: Tuple[str] = ('retry', 'timeout', 'metadata')
+
+    METHOD_TO_PARAMS: Dict[str, Tuple[str]] = {
+        {% for method in service.methods.values() -%}
+          '{{ method.name|snake_case }}': ({% for field in method.legacy_flattened_fields.values() %}'{{ field.name }}', {% endfor %}),
+        {% endfor -%}
+    }
+
+    def leave_Call(self, original: cst.Call, updated: cst.Call) -> cst.CSTNode:
```

Comment:
The casing is a bit weird. Did you mean `leaveCall` or `leave_call`?

Reply:
I mean leave_Call. It's the naming from libCST, and this is a method override as a child class of cst.CSTVisitor.

https://libcst.readthedocs.io/en/latest/visitors.html#visit-and-leave-helper-functions
https://libcst.readthedocs.io/en/latest/tutorial.html#Build-Visitor-or-Transformer